src/AddTodoForm.js

```diff
@@ -1,14 +1,20 @@
 import React from 'react';
 
-const AddTodoForm = () => {
-    return(
+const AddTodoForm = (props) => {
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        let todoTitle = event.target.value;
+        console.log(todoTitle);
+        onAddTodo(todoTitle);
+    }
+    return( 
     <div>
         <form>
             <label htmlFor='todoTitle'> TITLE </label> <br /> <br />
-            <input type='text' id='todoTitle'/>  <br /> <br />
+            <input onChange={handleAddTodo} type='text' id='todoTitle' name='title' />  <br /> <br />
```

Comment:
We're not using `onChange` event in this week's assignment, you should use `onSubmit` event instead (hint: that last event is not used on a type text input)